packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,84 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  --terra-consumer-color: #64696c;
+  --terra-consumer-color-dark-30: #383f42;
+  --terra-consumer-color-dark-50: #1c1f21;
+  --terra-consumer-color-light-30: rgba(#383f42, 0.3);
+  --terra-consumer-color-light-50: rgba(#64696c, 0.5);
+  --terra-consumer-color-light-65: rgba(#383f42, 0.65);
+  @include terra-consumer-theme-button();
+
+  /* stylelint-disable selector-class-pattern */
+  /* stylelint-disable suitcss/custom-property-no-outside-root */
+
+  .terra-consumer-theme-concept1 {
+    --terra-consumer-color: #c24726;
+    --terra-consumer-color-dark-30: #97371e;
+    --terra-consumer-color-dark-50: #6c2715;
+    --terra-consumer-color-light-30: rgba(#c24726, 0.3);
+    --terra-consumer-color-light-50: rgba(#c24726, 0.5);
+    --terra-consumer-color-light-65: rgba(#c24726, 0.65);
+    @include terra-consumer-theme-button();
```

Comment:
assuming `.terra-consumer-theme-concept1` is always nested in `cerner-consumer-theme` this shouldn't be needed.